src/lib/fragments/CookiePages/Cookies.styles.js

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+import styled from 'styled-components';
+import{ ComponentType } from 'react';
+
+
+export const Article = (() =>
+    styled.article`
+        max-width: 50em;
+    `
+)();
+
+
+export const Heading = ({ className = '', ...props }) => (
```

Comment:
Can this be in a typography tsx and then it's reusable to gov standards. It good for DSL to have dedicated typography file ans components

Reply:
We've created a ticket to build a typography file.